### PR TITLE
feat: add `use_clyde_ai` permission and role subscription purchase notifications to `SystemChannelFlags`

### PR DESCRIPTION
**Pull request by MCausc78**
https://github.com/discord/discord-api-docs/pull/6354

## Information

- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed).
- [ ] This PR is **not** a code change (e.g. documentation, README, typehinting,
      examples, ...).

## Checklist

- [x] I have searched the open pull requests for duplicates.
- [x] If code changes were made then they have been tested.
  - [x] I have updated the documentation to reflect the changes.
- [ ] If `type: ignore` comments were used, a comment is also left explaining why.
- [x] I have updated the changelog to include these changes.

### CODE REVIEW

**Comment by Lulalaby** [comment]
Well while ur on it, please add SetVoiceChannelStatus
```cs
/// <summary>
/// Allows members to create and edit voice channel status.
/// </summary>
[PermissionString("Set Voice Channel Status"), DiscordInExperiment]
SetVoiceChannelStatus = 1L << 48
```

And look for other possible missing permissions (i think the create_xy are missing)

File: CHANGELOG.md [hunk 1]
```diff
@@ -75,6 +75,9 @@ These changes are available on the `master` branch, but have not yet been releas
   ([#2206](https://github.com/Pycord-Development/pycord/pull/2206))
 - Added function `Guild.delete_auto_moderation_rule`.
   ([#2153](https://github.com/Pycord-Development/pycord/pull/2153))
+- Added
+  [`USE_CLYDE_AI` permission](https://github.com/discord/discord-api-docs/pull/6354).
```
Comment (EmreTech) [inline CHANGELOG.md:79]: What about the role subscription purchase notifications flag?

Comment (Lulalaby) [reply CHANGELOG.md:79]: ^ + the changelog entry is entirely wrong

**Comment by plun1331** [comment]
@MCausc78 any updates?

**Comment by Dorukyum** [comment]
@MCausc78 Clyde has been deactivated, please update the pull request accordingly.

**Pull request closed without merging**